openvdb_houdini/abitest/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+# Copyright Contributors to the OpenVDB Project
+# SPDX-License-Identifier: MPL-2.0
+#
+#[=======================================================================[
+
+  CMake Configuration for the OpenVDB Houdini ABI Test
+
+  This test verifies ABI compatibility by constructing VDB objects using
+  this version of VDB and then validating using the version of VDB shipped
+  with Houdini and vice-versa. This assumes that the ABI between the two
+  is the same.
+
+#]=======================================================================]
+
+cmake_minimum_required(VERSION 3.3)
+project(OpenVDBHoudiniABITest)
+
+# Monitoring <PackageName>_ROOT variables
+if(POLICY CMP0074)
+  cmake_policy(SET CMP0074 NEW)
+endif()
+
+#########################################################################
+
+message(STATUS "----------------------------------------------------")
+message(STATUS "-------- Configuring OpenVDBHoudiniABITest ---------")
+message(STATUS "----------------------------------------------------")
+
+##########################################################################
+
+# This binary is composed of two object files:
+# - TestABI uses headers from this version of OpenVDB
+# - main uses headers from the version of OpenVDB shipped with Houdini
+
+# Disable some compiler warnings in TBB and Half
+
+add_compile_options(
+  -Wno-conversion
+  -Wno-shorten-64-to-32
+)
```

Comment:
Hmm you shouldn't be getting these, cmake should treat these projects as system headers?

Reply:
I'll try and dig into what's going on here a bit.

Reply:
The `target_include_directories` was missing the SYSTEM keyword. Fixed.